app/src/components/dialog/YesNoDialog.tsx

```diff
@@ -69,14 +108,22 @@ const YesNoDialog: React.FC<IYesNoDialogProps> = (props) => {
       aria-describedby="alert-dialog-description">
       <DialogTitle id="alert-dialog-title">{props.dialogTitle}</DialogTitle>
       <DialogContent>
-        <DialogContentText id="alert-dialog-description">{props.dialogText}</DialogContentText>
+        {props.dialogText && <DialogContentText id="alert-dialog-description">{props.dialogText}</DialogContentText>}
+        {props.dialogContent}
       </DialogContent>
       <DialogActions>
-        <Button data-testid="yes-button" onClick={props.onYes} color="primary" variant="contained" autoFocus>
-          Yes
+        <Button
+          data-testid="yes-button"
+          onClick={props.onYes}
+          color="primary"
+          variant="contained"
+          autoFocus
+          {...props.yesButtonProps}>
+          {props.yesButtonLabel ? props.yesButtonLabel : 'Yes'}
         </Button>
+
         <Button data-testid="no-button" onClick={props.onNo} color="primary" variant="outlined">
```

Comment:
Can you add the noButtonProps at the end of this button, like with the yes button